src/ralph/utils.py

```diff
@@ -84,3 +84,16 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def nested_get(dictionary, keys):
```

Comment:
Secure the implementation by using type hints!

```suggestion
def nested_get(dict_: dict, keys: list[str]):
```

Reply:
Looks much better this way! Thanks)